SpatialGDK/Documentation/content/generating-a-snapshot.md

```diff
@@ -47,7 +47,7 @@ If you want your snapshots to be exported to a different path you can specify th
 ## When to generate a snapshot
 You need to regenerate a snapshot when:
```

Comment:
Point of clarity - just realised you generate a snapshot _after_ you have done these, not during (as suggested by present continuous verb (ing) here).
```suggestion
You need to regenerate a snapshot when you have:
```